server/bin/test-bin/ssh

```diff
@@ -9,5 +9,8 @@ if [ "${remote_host}" = "pbench-satellite.example.com" ] ;then
     # environment from leaking into the "remote" server environment.
     unset _PBENCH_SERVER_CONFIG
     $*
+else
```

Comment:
I think this mock might be used by other tests eventually.  You might want to check for a specific `${remote_host}` value that would trigger the error.

Reply:
I did use the value 'pbench-satellite-error.example.com' I assigned that in the config file

Reply:
Yes, but it would seem you need to check for that value to take this action only when that value is provided.  That way, another test won't get tripped up by that behavior.

Reply:
ok